containers/documentation.docker/build_documentation

```diff
@@ -17,6 +17,13 @@
 
 # get script dir
 script_dir="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+# determine if this is run in Cygwin under Windows
+is_cygwin=false
+if test "$(uname -o||true)" == "Cygwin"; then
```

Comment:
[[ "$(uname -o || true)" == "Cygwin" ]]

Reply:
We rely heavily on bash we want to use `[[ ]]`. We use exported functions and a lot of other stuff that does not work with other shells.